examples/src/lib.rs

```diff
@@ -1 +1 @@
-
+//! Examples on using Zenoh. Check the Readme for usage.
```

Comment:
For even more clarity I would reword as such:
```rust
//! Examples on using Zenoh. 
//! See the code in ../examples/ 
//! Check ../README.md for usage.
```

Reply:
I agree.